UNRELEASED.md

```diff
@@ -1,5 +1,13 @@
 # Unreleased
 
+**Breaking**:
+  - `BpkIcon`:
+    - Renamed `bpk_swap` to `bpk_swap__horizontal`:
```

Comment:
It seems like the `bpk_swap` icon was not removed, can you try running `npm run build`

Reply:
`npm run build` didn't remove anything. Where are you seeing it persisting?

Reply:
if `bpk-swap` was removed then you need to remove `Backpack/src/main/res/drawable-nodpi/bpk_swap.xml` too. I guess this is a limitation of our script.

Reply:
I already did tho
https://github.com/Skyscanner/backpack-android/pull/317/files#diff-77a61cb12bc52ad8ed6d931f331f9ecd

Reply:
Githubs showing it as a rename instead of delete because it recognises the content is the same...

Reply:
I right, sorry I didn't see it